server/bin/unittests

```diff
@@ -49,6 +49,8 @@ function _run {
     else
         echo "+++ Running ${tname} ${@}" >> ${_testout}
     fi
+    export PYTHONPATH=${_testopt}/lib:$PYTHONPATH
+    export PYTHONWARNINGS="ignore::DeprecationWarning:configparser"
```

Comment:
So we need this because ... ?

Reply:
Because they are removing the older version of configparser which is throwing warning that, "change your configparser into the latest one if you are using the older version else it will create problem in the future"

Reply:
@ndokos, does https://github.com/distributed-system-analysis/configtools/pull/16 fix this?

Reply:
Yes, it does.